examples/todo/src/models/todo-with-id.model.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright IBM Corp. 2018,2019. All Rights Reserved.
+// Node module: @loopback/example-todo
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Entity, model, property} from '@loopback/repository';
+
+@model()
+export class TodoWithId extends Entity {
+  @property({
+    type: 'number',
+    id: true,
+    required: true,
+  })
+  id?: number;
```

Comment:
Since `id` is `required: true`, I think we should tell TypeScript to make it require too?

```suggestion
  id: number;
```